include/lsst/afw/table/Schema.h

```diff
@@ -417,7 +417,7 @@ class SubSchema {
     /**
      *  @brief Implicit conversion to the appropriate Key type.
      *
-     *  Implicit conversion operators cannot be translated to Python.  Instead, the SWIG
+     *  Implicit conversion operators cannot be translated to Python.  Instead, the Python
      *  wrappers provide an equivalent asKey() method.
      */
     template <typename T>
```

Comment:
The kind we need here - a conversion invoked by assignment - cannot be replicated in Python, regardless of the wrapper tool used.